### PR TITLE
Replace ATLAS logo (Closes #788)

### LINKED ISSUE #788: Update ATLAS logo
Replace

> http://opendata.cern.ch/img/ATLAS.gif

with

> [EPS] http://atlas-outreach.web.cern.ch/sites/atlas-outreach.web.cern.ch/files/ATLAS-Logo-Square-Blue-RGB.eps

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
As requested by ATLAS.

### CODE REVIEW

**Comment by jirikuncar** [comment]
@RaoOfPhysics do you have a SVG version of ATLAS logo?

**Comment by RaoOfPhysics** [comment]
@jirikuncar, EPS here: http://atlas-outreach.web.cern.ch/sites/atlas-outreach.web.cern.ch/files/ATLAS-Logo-Square-Blue-RGB.eps

**Comment by jirikuncar** [comment]
@RaoOfPhysics well, I don't think many browsers can include EPS graphics, but they can handle SVG. We can add it later ...

**Comment by RaoOfPhysics** [comment]
@jirikuncar: I meant you could use the EPS file to generate an SVG. I made a GIF simply because the previous one was the same format.

**Comment by TimSmithCH** [comment]
Ok, I launched AI and made this SVG:
https://cernbox.cern.ch/public.php?service=files&t=13762c15cff68eb275f1e08398cfa90a

**Comment by RaoOfPhysics** [comment]
@TimSmithCH: Thanks! I made one last night after pinging @jirikuncar, but hadn't had the time to update the PR.

**Question**: Should I also replace all the links that point, currently, to `ATLAS.gif` with the SVG filename? I searched in the project and I could only find two location that pointed to it:

- https://github.com/cernopendata/opendata.cern.ch/blob/d28182b6c326ff7be1408686bd368e22d06d12dc/invenio_opendata/base/fixtures/websearch.py#L572
- https://github.com/cernopendata/opendata.cern.ch/blob/d28182b6c326ff7be1408686bd368e22d06d12dc/invenio_opendata/base/templates/footer.html#L48

**Comment by RaoOfPhysics** [comment]
@jirikuncar, @TimSmithCH: Let me know if more changes need to be made! :)

**Comment by jirikuncar** [comment]
@RaoOfPhysics can you amend https://github.com/cernopendata/opendata.cern.ch/blob/d28182b6c326ff7be1408686bd368e22d06d12dc/invenio_opendata/base/templates/footer.html#L48 and set proper dimensions of the svg image? Thanks

File: invenio_opendata/base/static/img/ATLAS-Logo-Square-Blue-RGB.svg [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jirikuncar) [inline invenio_opendata/base/static/img/ATLAS-Logo-Square-Blue-RGB.svg]: Can you rename this file to `ATLAS.svg` to keep consistency with other logos?

**Comment by RaoOfPhysics** [comment]
@jirikuncar: Not sure how I should resize the SVG. 

```html
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="259px" height="259px" viewBox="0 0 259 259" enable-background="new 0 0 259 259" xml:space="preserve">
```

**Comment by jirikuncar** [comment]
@RaoOfPhysics what I meant was to set style to the ` ` with ATLAS logo (something like: `` or even better set it in global css).

**Comment by RaoOfPhysics** [comment]
@jirikuncar: Terribly sorry, this isn't my strength, and I didn't do any of the website design. I don't want to change any of the backend that @tiborsimko and @pamfilos built.

**Comment by tiborsimko** [comment]
> set proper dimensions of the svg image?

We don't need to amend width/height of logos in the page footer, this is already handled by the CSS.

>  made this SVG

It's good to use SVG in the page footer, but we actually need GIF as well, because of the way how the experiment logo image paths are "dynamically" constructed and loaded in the page breadcrumb trail and on the collection pages.

Thanks, squashed together, added GIF, amended commit log message, and merged.

**Pull request closed without merging**